scripts/post.js

```diff
@@ -332,6 +332,71 @@ function decorateImages() {
   })
 }
 
+/**
+ * Loops through a list of keywords and looks for matches in the article text.
+ * The first occurrence of each keyword will be replaced with a link.
+ */
+async function addInterLinks() {
+  const response = await fetch('/en/keywords.json');
+  if (response.ok) { 
+    const json = await response.json();
+    let bStart = Date.now();
+    const keywords = (Array.isArray(json) ? json : json.data)
+      // scan article to filter keywords down to relevant ones
+      .filter(({ Keyword }) => document.querySelector('main').textContent.toLowerCase().indexOf(Keyword) !== -1);
+    console.log(`keywords: filtering down from ${json.length} to ${keywords.length} took ${Date.now() - bStart}ms`);
```

Comment:
You need to remove the debugging `console.log` at one point.

Reply:
Yes of course. Just keeping them in now for convenience while we're still testing on various articles and edge cases